File: HomeWork1/src/una/cpp/Person.cpp
#include "../Person.h"
#include <sstream>

Person::Person(string id, string name, string middleName, string lastName, string birthDate, string salary) {
    this->id = id;
    this->name = name;
    this->middleName = middleName;
    this->lastName = lastName;
    this->birthDate = birthDate;
    this->salary = salary;
}

string Person::getId() const
{
    return id;
}

string Person::getName() const
{
    return name;
}

string Person::getMiddleName() const
{
    return middleName;
}

string Person::getLastName() const
{
    return lastName;
}

string Person::getBirthDate() const
{
    return birthDate;
}

string Person::getSalary() const
{
    return salary;
}

string Person::toString()
{
    stringstream s;
    s << id + " - " + name + " " + middleName + " " + lastName + " - " + birthDate + " - " + salary;
    return s.str();
}

Person::~Person()
{
}

File: HomeWork2/src/main/java/cr/una/Person.java
package cr.una;

public class Person {
    private String id;
    private String middleName;
    private String lastName;
    private String name;
    private String birthDate;
    private double salary;
    private double deductions;
    private double netSalary;

    public Person() {
    }

    public Person(String id, String middleName, String lastName, String name, String birthDate, double salary) {
        this.id = id;
        this.middleName = middleName;
        this.lastName = lastName;
        this.name = name;
        this.birthDate = birthDate;
        this.salary = salary;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getMiddleName() {
        return middleName;
    }

    public void setMiddleName(String middleName) {
        this.middleName = middleName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(String birthDate) {
        this.birthDate = birthDate;
    }

    public double getSalary() {
        return salary;
    }

    public void setSalary(double salary) {
        this.salary = salary;
    }

    public double getDeductions() {
        return deductions;
    }

    public void setDeductions(double deductions) {
        this.deductions = deductions;
    }

    public double getNetSalary() {
        return netSalary;
    }

    public void setNetSalary(double netSalary) {
        this.netSalary = netSalary;
    }

    @Override
    public String toString() {
        return "Person{" +
                "id='" + id + '\'' +
                ", middleName='" + middleName + '\'' +
                ", lastName='" + lastName + '\'' +
                ", name='" + name + '\'' +
                ", birthDate='" + birthDate + '\'' +
                ", salary=" + salary +
                ", deductions=" + deductions +
                ", netSalary=" + netSalary +
                '}';
    }
}
